client/blocks/edit-gravatar/index.jsx

```diff
@@ -158,8 +159,10 @@ export class EditGravatar extends Component {
 	render() {
 		const {
 			isUploading,
+			translate,
 			user
 		} = this.props;
+		const gravatarLink = `//gravatar.com/${ user.username || '' }`;
```

Comment:
we should probably just force this to `https`. Gravatar supports it so there's no great reason to downgrade to http.